src/migrations/M00001.js

```diff
@@ -21,15 +21,15 @@ exports.up = function (knex) {
 					console.log(
 						'Unable to locate config.json! Things may fail unexpectedly!',
 					);
-					resolve();
+					resolve(false);
```

Comment:
I added these because the typescript compiler was complaining with 

```
src/migrations/M00002.js:20:12 - error TS2794: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?

20     return resolve();
              ~~~~~~~~~

  node_modules/typescript/lib/lib.es2015.promise.d.ts:33:34
    33     new <T>(executor: (resolve: (value: T | PromiseLike<T>) => void, reject: (reason?: any) => void) => void): Promise<T>;
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~
    An argument for 'value' was not provided.
```